pyclam/manifold.py

```diff
@@ -262,11 +261,15 @@ def descendents(self) -> List['Cluster']:
     @property
     def ratios(self) -> List[float]:
         if 'ratios' not in self.cache:
-            self.cache['ratios'] = [  # Child/Parent Ratios
-                self.local_fractal_dimension / self.parent.local_fractal_dimension,  # local fractal dimension
-                self.cardinality / self.parent.cardinality,  # cardinality
-                max(self.radius, 1e-16) / max(self.parent.radius, 1e-16)  # radius
-            ] if self.depth > 0 else [1., 1., 1.]
+            if self.depth > 0:
+                lfd_ratio = self.local_fractal_dimension / self.parent.local_fractal_dimension
+                self.cache['ratios'] = [  # Child/Parent Ratios
+                    1 / (1 + np.exp(-lfd_ratio)),  # local fractal dimension
```

Comment:
interesting to just do this for one ratio

Reply:
doing this significantly improved the quality of fit of the linear regression models. The other ratios don't need this